src/devices/Hts221/samples/README.md

```diff
@@ -1,4 +1,4 @@
-﻿# HTS221 - Capacitive digital sensor for relative humidity and temperature
+# HTS221 - Capacitive digital sensor for relative humidity and temperature
```

Comment:
I believe encoding changed in this file. Please revert to original

Reply:
Done, apologies these got inexplicably changed by Gitpod. Fixed.